cmd/apiserver/app/server/run_server.go

```diff
@@ -42,7 +42,8 @@ func RunServer(opts *ServiceCatalogServerOptions) error {
 func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	tprOpts := opts.TPROptions
 	glog.Infoln("Installing TPR types to the cluster")
-	if err := tpr.InstallTypes(tprOpts.clIface); err != nil {
+	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
```

Comment:
I'm still confused why we need a new type to hold the TPR interface. why not just have an `InstallTypes` function and pass that interface as a param?